### PR TITLE
Demo notebook + lots of cleanup

### LINKED ISSUE #5: Implement demo inference script
Refer to the file `run_model.py`. The script should take a pre-trained model and setup a live webcam feed to test mask detection. (optionally add a CLI flag to forgo webcam setup in favor of running the model on a passed image file instead)

Reference repos / scripts

- https://github.com/iamirmasoud/facial_keypoint_detection/blob/main/5_Webcam.ipynb
- https://github.com/e-lab/pytorch-demos/blob/master/face-keypoint/README.md

### PR DESCRIPTION
**Pull request by Toalaah**
closes #5

### CODE REVIEW

**Review by EnnioEnnio — COMMENTED** [review]
Awesome! Looks very good to me! 
One minor question regarding data-preprocessing: We do not support manual preprocessing with rotation etc any more, right? Or am I missing it in the data-loader? As far as I saw it it's just resizing and reshaping to match the architecture. (Would be totally fine for me)

**Comment by Toalaah** [comment]
Yes, the dataset is already pre-processed, making the script a little pointless. Especially since the `ImageFolder` already applies the transformations (and rotations if you want, although we currently don't add any), you don't even need to resize the images beforehand. It is done on the fly by the `DataLoader`.

**Comment by Toalaah** [comment]
Yep, it' not in the readme anymore. I will go ahead and merge this then, seems there are no more concerns. If there are, a quick fixup MR is always a possibility.
